cfgov/unprocessed/css/molecules/image-text-50-50.less

```diff
@@ -82,7 +82,6 @@
 */
 
 .m-image-text-50-50 {
-    padding-bottom: unit( @grid_gutter-width * 2 / @base-font-size-px, em );
```

Comment:
I removed that line and added the spacing on the `.m-image-text-50-50-group` element to avoid the need to use nth-of-type or not.